MapboxCoreNavigation/RouteController.swift

```diff
@@ -843,6 +890,20 @@ extension RouteController {
 
         return event.id.uuidString
     }
+    
+    func enqueueFoundFasterRouteEvent() -> String {
+        let timestamp = Date()
+        let eventDictionary = events.navigationRerouteEvent(routeController: self, MMEEventType: FasterRouteFoundEvent)
+        
+        sessionState.lastRerouteDate = timestamp
+        sessionState.numberOfReroutes += 1
```

Comment:
We shouldn't increment either of these (`numberOfReroutes` and `lastRerouteDate`) values, since a faster reroute is intentional, and those are meant to track problematic reroutes.